.github/workflows/backend-ci.yml

```diff
@@ -31,9 +31,13 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
+          registry-url: https://npm.pkg.github.com
+          scope: '@nearform'
 
       - name: Installing dependencies
         working-directory: packages/titus-backend
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.BROKENECK_NPM_TOKEN }}
```

Comment:
these changes are necessary to install the brokeneck packages from the github registry, since brokeneck is a private repository currently